www/js/search.js

```diff
@@ -727,31 +739,40 @@ module.exports = {
 
   lineFactory(line, rows) {
     const mainLineExists = !!document.querySelector('.main');
+    const englishHeadingEl = document.createElement('span');
+    englishHeadingEl.innerHTML = line.English;
```

Comment:
need to do something about the tags on chromecast as well.
![image](https://user-images.githubusercontent.com/2982784/57714220-e7214f00-7628-11e9-86ed-e9c030beee25.png)